sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -857,9 +857,9 @@ def encode(self, table_row):
     # to the programmer that they have used NAN/INF values.
     try:
       return json.dumps(
-          table_row, allow_nan=False, default=default_encoder)
+          table_row, allow_nan=False, default=default_encoder).encode('utf-8')
     except ValueError as e:
       raise ValueError('%s. %s' % (e, JSON_COMPLIANCE_ERROR))
 
   def decode(self, encoded_table_row):
-    return json.loads(encoded_table_row)
+    return json.loads(encoded_table_row.decode('utf-8'))
```

Comment:
Could you please check that this works with Tornadoes example on DirectRunner?

Reply:
Yes, this continues to work with both DirectRunner and DataflowRunner.